docs/troubleshooting/resharding.md

```diff
@@ -60,6 +60,16 @@ If you observe problems with block production or resharding performance, you can
 This does not require a node restart, you can send a signal to the neard process to load the new config.
 Read more [on github](https://github.com/near/nearcore/blob/master/docs/architecture/how/resharding.md#monitoring).
 
+### Mitigating state snapshot issue {#state snapshot}
+Node has to have a state snapshot in order for resharding to run.
+State snapshot is a smaller checkpoint of the whole DB taken at the epoch boundary.
```

Comment:
I would also add something like:
"If the node fails to capture a snapshot at the epoch boundary it will not be able to proceed with resharding. In this case manual recovery will be needed."